### PR TITLE
Allow to Set Different Certificate Print Format for each LMS Course

### PR DESCRIPTION
**Pull request by Rola-anvil**
This is to allow the system to create different certificates for each course

### CODE REVIEW

**Comment by ahmadRagheb** [comment]
nice job rola

**Comment by pateljannat** [comment]
@Rola-anvil I don't understand the need for a link field in the LMS Course doctype.

When creating a certificate, it considers the default print format set.

You can just make your desired format as default.

**Comment by Rola-anvil** [comment]
But Every course needs a different certificate design, so I have created this field to let the system user choose a custom certificate design or  consider the default one

File: lms/fixtures/custom_field.json [hunk 1]
```diff
@@ -2012,5 +2012,58 @@
...
+    "bold": 0,
+    "collapsible": 0,
+    "collapsible_depends_on": null,
+    "columns": 0,
+    "default": null,
+    "depends_on": null,
+    "description": null,
+    "docstatus": 0,
+    "doctype": "Custom Field",
+    "dt": "LMS Course",
```
Comment (pateljannat) [inline lms/fixtures/custom_field.json:2028]: @Rola-anvil why add a custom field in LMS Course doctype when you can add a standard field.

**Review by pateljannat — CHANGES_REQUESTED** [review]

File: lms/fixtures/custom_field.json (hunk 1 above)
Comment (pateljannat) [reply lms/fixtures/custom_field.json:2028]: Also, you have made the changes in the main branch of your fork. Can you create another branch for this feature instead?

**Pull request closed without merging**